process/Cargo.toml

```diff
@@ -4,6 +4,6 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-sysinfo = "*"
+sysinfo = "0.30"
```

Comment:
1) Are this changes in "process" related to this PR/parameters?
2) why the "sysinfo" package version change?

Reply:
Yes, because the `*`, CI picked up a different older version than what I was building on my local system so some of the syntax changed between versions.  Having explicit version keeps it consistent so it builds both locally and in CI.